src/GlobalScreen/ScreenContext/BasicScreenContext.php

```diff
@@ -95,10 +95,23 @@ public function getAdditionalData() : Collection
     public function addAdditionalData(string $key, $value) : ScreenContext
     {
         $this->additional_data->add($key, $value);
-
+   
         return $this;
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function addAdditionalDataIfNotExist(string $key, $value) : ScreenContext
+    {
+        if($this->additional_data->exists($key)) {
```

Comment:
Hi @skullbox305,

please use the `php-cs-fixer` to apply our coding style guideline (see: https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/coding-style.md).

A detailed review will be provided by the assignees.

Best regards,
@mjansenDatabay